wallpanel.js

```diff
@@ -1573,11 +1587,48 @@ class WallpanelView extends HuiView {
 		http.send();
 	}
 
+	updateImageListFromCustomAPI(preload) {
+		this.updatingImageList = true;
+		this.lastImageListUpdate = Date.now();
+		let wp = this;
+		let urls = [];
+		let data = {};
+		const http = new XMLHttpRequest();
+		http.responseType = "json";
+		
+		let custom_api_url = config.image_url.replace('custom-api-', '');
+		http.open("GET", `${custom_api_url}`, true);
+		http.onload = function() {
+			if (http.status == 200 || http.status === 0) {
+				if (config.debug) console.debug(`Got custom API response`);
+				http.response.forEach(entry => {
+					if (config.debug) console.debug(entry);
+					const url = entry.url;
+					urls.push(url);
+					data[url] = entry;
```

Comment:
I think it would be better to use a sub key for the imageInfo.
```suggestion
					data[url] = entry.imageInfo;
```